packages/editor/src/store/utils/notice-builder.js

```diff
@@ -108,10 +108,11 @@ export function getNotificationArgumentsForSaveFail( data ) {
 	// Check if message string contains HTML. Notice text is currently only
 	// supported as plaintext, and stripping the tags may muddle the meaning.
 	if ( error.message && ! ( /<\/?[^>]*>/.test( error.message ) ) ) {
-		noticeMessage = sprintf( __( '%1$s Error message: %2$s' ), noticeMessage, error.message );
+		noticeMessage = [ noticeMessage, error.message ].join( ' ' );
 	}
 	return [ noticeMessage, {
 		id: SAVE_POST_NOTICE_ID,
+		type: 'snackbar',
```

Comment:
I'll add some "design" checks here but the idea was that errors need to be more prominent notices and not just snackbars you can miss.